### PR TITLE
add gRPC to templates

### PR DESCRIPTION
**Pull request by jennyf19**
Testing todo:
- [x] dotnet pack in ProjectTemplates
- [x] dotnet -u in bin\Debug to remove the previous templates
- [x] dotnet -i in bin\Debug to re-add the new templates
- [x] create a sub folder
- [x] dotnet new worker2 and see what this gives (without options)
- [ ] With all the auth options 
  - [x] --auth SingleOrg
  - [x] --auth  IndividualB2C
  - [x] --auth None
  - [x] nothing
	
FYI: @jmprieur

### CODE REVIEW

**Comment by jennyf19** [comment]
@jmprieur having issues w/the no auth option, unless i mark it was `--auth None`, but just doing `dotnet new worker2` doesn't find the template.

File: ProjectTemplates/templates/Worker-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Worker",
+    "Web"
+  ],
+  "name": "Worker Service",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating a worker service.",
+  "groupIdentity": "Microsoft.Worker.Empty",
```
Comment (jmprieur) [inline ProjectTemplates/templates/Worker-CSharp/.template.config/template.json:12]: you probably want to rename the groupIdentity to "Microsoft.Worker.Empty"

File: ProjectTemplates/templates/Worker-CSharp/.template.config/template.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Worker",
+    "Web"
+  ],
+  "name": "Worker Service",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating a worker service.",
```
Comment (jmprieur) [inline ProjectTemplates/templates/Worker-CSharp/.template.config/template.json:11]: An empty project template for creating a worker service which supports authentication with Azure AD and Azure AD B2C  ?

**Review by jmprieur — APPROVED** [review]
LGTM. Great job @jennyf19 
I've made two proposals.

**Comment by jennyf19** [comment]
that fixed it. thanks.

---
In reply to: [758351238](https://github.com/AzureAD/microsoft-identity-web/pull/859#issuecomment-758351238) [](ancestors = 758351238)

**Comment by jennyf19** [comment]
thank you. fixed

---
In reply to: [566285789](https://github.com/AzureAD/microsoft-identity-web/pull/859#issuecomment-566285789) [](ancestors = 566285789)

**Pull request merged**